### PR TITLE
Reward - Using Kudos

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/UsingKudos.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+############
+Using Kudos
+############
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos in order to thank them for their help, contributions and good work.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
You can express your recognition to your colleagues by sending them kudos in order to thank them for their help, contributions or good work.
```

File: docs/UsingKudos.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+You have a limited number of kudos that you are allowed to send per period of time.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
You have a limited number of kudos to send per period of time. 
```

File: docs/UsingKudos.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos in order to thank them for their help, contributions and good work.
+You have a limited number of kudos that you are allowed to send per period of time. 
+
+If you don't use the kudos during the current period, you will not be able to use them in the next one. They are not cumulated with the number of kudos you are allowed to use in the new period.
+Both the number of kudos and periodicity are determined by the administrator of the platform.
+
+When you send a kudos you earn gamification points as well as the reciever.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion

```
This depends of a gamification rule that can be disabled. I would not mention it here.

File: docs/UsingKudos.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos in order to thank them for their help, contributions and good work.
+You have a limited number of kudos that you are allowed to send per period of time. 
+
+If you don't use the kudos during the current period, you will not be able to use them in the next one. They are not cumulated with the number of kudos you are allowed to use in the new period.
+Both the number of kudos and periodicity are determined by the administrator of the platform.
+
+When you send a kudos you earn gamification points as well as the reciever. 
+The reciever may beneit of an amount of cauris thank to the recieved kudos if the administrator has configured the reward sending at the reception of the kudos.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
The receiver may be rewarded for the kudos he received if the administrator has enabled and configured the Kudos rewarding program. 
```

File: docs/UsingKudos.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+Both the number of kudos and periodicity are determined by the administrator of the platform.
+
+When you send a kudos you earn gamification points as well as the reciever. 
+The reciever may beneit of an amount of cauris thank to the recieved kudos if the administrator has configured the reward sending at the reception of the kudos. 
+
+
+How to send kudos
+~~~~~~~~~~~~~~~~~~
+
+You can send kudos from the following emplacements of the platform:
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
You can send kudos from the following locations:
```

File: docs/UsingKudos.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+- The user's profile
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
- The user profile
```

File: docs/UsingKudos.rst [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+- The popover of user from different activities
+
+|image1|
+
+- An activity posted by the user
+
+|image2|
+
+- A comment written by the user
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
- A comment posted by the user 
```

File: docs/UsingKudos.rst [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+- An activity posted by the user
+
+|image2|
+
+- A comment written by the user 
+
+|image3|
+
+When you click on the icon to send a kudos, a popover is displayed containing the details about the already sent kudos in the current period with and the left ones.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
When you click on the icon to send a kudos, a popover is displayed containing the details of the kudos you already sent during the current period wand the number of remaining kudos.
```

File: docs/UsingKudos.rst [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+You have to type in a text to explain the reason of this action:
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
When sending a kudos, you must leave a message so you can tell the reason for this kudos:
```

File: docs/UsingKudos.rst [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+|image4|
+
+If you have no more kudos left for the current period, a warning message will be displayed in the popover as well as the information about the kudos you already sent.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
If you have no  kudos left for the current period, a warning message will be displayed in the popover as well as the information about the kudos you already sent.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
proofreading : suggestions of simplifications and some language fixes

File: docs/UsingKudos.rst (hunk 3 above)
Comment (samehaidi) [reply docs/UsingKudos.rst]: Ok. I deleted it.

File: docs/UsingKudos.rst [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+############
+Using Kudos
+############
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos in order to thank them for their help, contributions or good work.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
You can express your recognition to your colleagues by sending them kudos when you want to thank them for their help, contributions or good work.
```

File: docs/UsingKudos.rst [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+You have a limited number of kudos to send per period of time.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
You have a limited number of kudos to send per period of time (week, month or quarter). 
```

File: docs/UsingKudos.rst [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+############
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos in order to thank them for their help, contributions or good work.
+You have a limited number of kudos to send per period of time. 
+
+If you don't use the kudos during the current period, you will not be able to use them in the next one. They are not cumulated with the number of kudos you are allowed to use in the new period.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
Kudos to send expire at the end of the period if you didn’t send them. Also, they can’t be cumulated with kudos you received. Thus, you can’t keep kudos for yourself they are give-only tokens of recognition.
```

File: docs/UsingKudos.rst [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+Both the number of kudos and periodicity are determined by the administrator of the platform.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
Both the number of kudos and periodicity of reset are defined by the administrator of the platform.
```

File: docs/UsingKudos.rst [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+
+The receiver may be rewarded for the kudos he received if the administrator has enabled and configured the Kudos rewarding program.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
When kudos are combined with their rewarding program, kudos received can be automatically converted into tokens of the eXo Wallet. 
```

File: docs/UsingKudos.rst [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+
+- An activity posted by the user
+
+|image2|
+
+- A comment posted by the user 
+
+|image3|
+
+When you click on the icon to send a kudos, a popover is displayed containing the details of the kudos you already sent during the current period and the number of remaining kudos.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
When you click on the icon to send a kudos, a popover is displayed containing the details of the kudos you already sent and the number of remaining kudos for the current period.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
worked on smoothness and rewrote paragraph that explains kudos to send vs to receive and reset period

File: docs/UsingKudos.rst [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+############
+Using Kudos
+############
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos when you want to thank them for their help, contributions or good work.
+You have a limited number of kudos to send per period of time (week, month, quarter or year).
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
You can send a limited number of kudos to per period of time (week, month, quarter or year). 
```

File: docs/UsingKudos.rst [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos when you want to thank them for their help, contributions or good work.
+You have a limited number of kudos to send per period of time (week, month, quarter or year). 
+
+Kudos to send expire at the end of the period if you didn’t send them. Also, they can’t be cumulated with kudos you received. Thus, you can’t keep kudos for yourself they are give-only tokens of recognition.
+Both the number of kudos and periodicity of reset are defined by the administrator of the platform.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
Both the number of kudos and period of reset are defined by the administrator of the platform.
```

File: docs/UsingKudos.rst [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+
+When kudos are combined with their rewarding program, kudos received can be automatically converted into tokens of the eXo Wallet.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
When kudos are combined with the rewarding program, kudos received can be automatically converted into tokens in the eXo Wallet. 
```

**Review by plamarque — CHANGES_REQUESTED** [review]
another pass of proofreading

File: docs/UsingKudos.rst [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+############
+Using Kudos
+############
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos when you want to thank them for their help, contributions or good work.
+You can send a limited number of kudos to per period of time (week, month, quarter or year).
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
You can send a limited number of kudos per period of time (week, month, quarter or year). 
```

File: docs/UsingKudos.rst [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+############
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos when you want to thank them for their help, contributions or good work.
+You can send a limited number of kudos to per period of time (week, month, quarter or year). 
+
+Kudos to send expire at the end of the period if you didn’t send them. Also, they can’t be cumulated with kudos you received. Thus, you can’t keep kudos for yourself they are give-only tokens of recognition.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
Kudos to send expire at the end of the period if you didn’t send them. Also, they can’t be cumulated with kudos you received. Thus, kudos are give-only tokens of recognition and you can’t keep them for yourself. You have to earned them from others.
```

File: docs/UsingKudos.rst [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+Both the number of kudos and period of reset are defined by the administrator of the platform.
+
+When kudos are combined with the rewarding program, kudos received can be automatically converted into tokens in the eXo Wallet.
```
Comment (plamarque) [inline docs/UsingKudos.rst]:
```suggestion
When kudos are combined with the rewarding program, they can be automatically converted into tokens in the eXo Wallet. 
```

**Review by plamarque — CHANGES_REQUESTED** [review]
Final pass...

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**